Gemfile

```diff
@@ -2,4 +2,5 @@ source "https://rubygems.org"
 
 git_source(:github) {|repo_name| "https://github.com/#{repo_name}" }
 
+gem "sneakers", github: "flatiron-labs/sneakers", branch: "upgrade-sneakers-to-2.11"
```

Comment:
PR should not go in until ironboard's sneakers dependency is reconciled. There's a world where these have to go in together.